server/services/store/sqlstore/blocks.go

```diff
@@ -634,6 +634,39 @@ func (s *SQLStore) getBlockHistory(db sq.BaseRunner, blockID string, opts model.
 	return s.blocksFromRows(rows)
 }
 
+func (s *SQLStore) getBlockHistoryDescendants(db sq.BaseRunner, boardID string, opts model.QueryBlockHistoryOptions) ([]model.Block, error) {
```

Comment:
@wiggin77, I reused the QueryBlockHistoryOptions struct here, because the intent is the same. Not sure if it was better to create a copy now, or do that later should they diverge.

Reply:
QueryBlockHistoryOptions makes sense here.